clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/AutoScalingWorker.groovy

```diff
@@ -217,6 +221,17 @@ class AutoScalingWorker {
     if (suspendedProcesses) {
       autoScaling.suspendProcesses(new SuspendProcessesRequest(autoScalingGroupName: asgName, scalingProcesses: suspendedProcesses))
     }
+    if (disabledGroupMetrics) {
```

Comment:
I don't really understand this bit - are there some metrics that are on by default that would need to be explicitly disabled right after ASG creation?